docker/3.9/Ubuntu/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04 AS builder-image
+
+# To avoid tzdata blocking the build with frontend questions
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --no-install-recommends -y python3.9 python3.9-dev python3.9-venv python3-pip python3-wheel build-essential && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# create and activate virtual environment
+RUN python3.9 -m venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+RUN pip3 install --no-cache-dir tiatoolbox
+
+FROM ubuntu:20.04 AS runner-image
```

Comment:
```suggestion
FROM ubuntu:22.04 AS runner-image?
```

Reply:
Done in [7f030d8](https://github.com/TissueImageAnalytics/tiatoolbox/pull/504/commits/7f030d821f83514aad5385703be701e856f583a4)